### DISCUSSION TITLE
CefSharp.BindObjectAsync got an error

### DISCUSSION THREAD

**Discussion by naphet** [cefsharp/CefSharp#3476] [General]
I invoke CefSharp.BindObjectAsync method,My Browser got an error, about C Runtime error。
The error description follows：
---------------------------
Microsoft Visual C++ Runtime Library
---------------------------
Runtime Error!

Program: C:\Us...

R6034

An application has made an attempt to load the C runtime library incorrectly.
Please contact the application's support team for more information.
---------------------------
OK   
---------------------------

**Comment by amaitland** [comment]
Please edit your original question and provide answers to the questions at https://github.com/cefsharp/CefSharp/wiki/Questions-&-Support#when-asking-a-question-please-include-the-following

What is the exact exception? Please include a stack trace. Please provide a code example.

**Reply by naphet** [reply]
I Fix My Program Bug, It is not a CefSharp Exception.
Program Load Assmbly From ms library then throw this exception.
thank u for reply

**Reply by naphet** [reply]
msvcm90.dll
